src/foundry/foundry.js/canvasAnimation.d.ts

```diff
@@ -4,14 +4,21 @@
 declare class CanvasAnimation {
   static get ticker(): PIXI.Ticker;
 
+  /**
+   * Track an object of active animations by name, context, and function
+   * This allows a currently playing animation to be referenced and terminated
+   */
+  static animations: Record<
+    string,
+    { fn: (dt: number) => void; context: PIXI.Container; resolve: (value: boolean) => void }
+  >;
```

Comment:
```suggestion
  static animations: Partial<
    Record<string, { fn: (dt: number) => void; context: PIXI.Container; resolve: (value: boolean) => void }>
  >;
```
This ensures that when accessing a member of `animations`, it's typed as `unknown | ...`